lib/templates.js

```diff
@@ -118,18 +118,13 @@ function _getTitleFromServiceName(srv) {
 const fCreateGroupsTemplateForService = (srv, srvDefinition) => {
     const ordExtensions = fReadORDExtensions(srvDefinition);
 
-  let fullyQualifiedServiceName = srv;
-  if (!srv.includes(global.capNamespace)) {
-    fullyQualifiedServiceName = global.capNamespace + "." + srv;
-  }
-
     if(!srvDefinition) {
       console.warn("Unable to find service definition:", srv)
       return undefined
     }
 
     if (srvDefinition && checkEntityFunctionAction(srvDefinition, global).length > 0) {
-      let groupId = _getGroupID(fullyQualifiedServiceName, defaults.groupTypeId);
+      let groupId = _getGroupID(srv, defaults.groupTypeId);
```

Comment:
What does `srv` mean here? Is the lastTwoFragments? I would suggest name it more context

Reply:
srv = service name; I've renamed it to the srvName? 
@zongqichen @Fannon @RamIndia : Sounds fine to you?

Reply:
`srv` is now `serviceName`, `srvDefinition` is `serviceDefinition` and `event` is `eventName`.